_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title:  "Introduction to system tests"
+date: 2020-09-07
+author: maroš_orsák
```

Comment:
Your should add your self to the authors file: https://github.com/strimzi/strimzi.github.io/blob/master/_data/authors.yml ... I think we never tried it with diacritics, so not sure how will that work.